packages/tools/client-debugger/client-debugger-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import {
+	tokens,
+	TableBody,
+	TableCell,
+	TableRow,
+	Table,
+	TableHeader,
+	TableHeaderCell,
+} from "@fluentui/react-components";
+import { Clock20Regular, DoorArrowLeft24Regular, Person24Regular } from "@fluentui/react-icons";
+import { FilteredAudienceHistoryData } from "./AudienceView";
+
+/**
+ * Input for {@link AudienceHistoryTable}
+ */
+export interface AudienceHistoryTableProps {
+	/**
+	 * Filtered audience data from {@link audienceHistoryDataFilter}
+	 * Containing clientId, timestamp & event.
+	 */
+	audienceHistoryItems: FilteredAudienceHistoryData[];
+}
+
+/**
+ * Render audience history in {@link AudienceView}
```

Comment:
Generally it's best to avoid documenting components in terms of what consumes them, since that can change. Let's instead document this in terms of what it displays.